src/renderer/qssWidget/js/qssOpenUsbWidget.js

```diff
@@ -0,0 +1,350 @@
+/**
+ * The QSS Open USB widget
+ *
+ * A custom widget for the Mount and Eject USB functionality
+ * used the menuWidget as a base and fires the onQssOpenUsbRequested and
+ * onQssUnmountUsbRequested events on the button clicks
+ *
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+/* global fluid */
+
+"use strict";
+(function (fluid) {
+    var gpii = fluid.registerNamespace("gpii");
+
+    /**
+     * Represents the QSS USB widget.
+     */
+    fluid.defaults("gpii.qssWidget.openUSB", {
+        gradeNames: ["fluid.viewComponent", "gpii.psp.heightObservable", "gpii.psp.selectorsTextRenderer"],
+        model: {
+            disabled: false,
+            setting: {},
+            messageChannel: "LoadInitialOfficeRibbonsState", // Channel listening for messages related to usb mount, unmount functionality
```

Comment:
Why are we listening to an OfficeRibbonsState channel when this is the USB widget?

Reply:
My mistake.